aws-datastore/src/androidTest/java/com/amplifyframework/datastore/BasicCloudSyncInstrumentationTest.java

```diff
@@ -60,16 +67,8 @@
  * which were defined by the schema in:
  * testmodels/src/main/java/com/amplifyframework/testmodels/commentsblog/schema.graphql.
  */
-@Ignore(
-    "Over time, this test will create a large DynamoDB table. Even if we delete the content " +
-    "through the AppSyncClient utility, the database will have lots of tombstone'd rows. " +
-    "These entries will be synced, the next time this test runs, and the DataStore initializes. " +
-    "After several runs, that sync will grow large and timeout the test, before the test can " +
-    "run any business logic. A manual workaround exists, by running this cleanup script: " +
-    "https://gist.github.com/jamesonwilliams/c76169676cb99c51d997ef0817eb9278#quikscript-to-clear-appsync-tables"
-)
 public final class BasicCloudSyncInstrumentationTest {
-    private static final int TIMEOUT_SECONDS = 10;
+    private static final int TIMEOUT_SECONDS = 30;
```

Comment:
Even with the sync expression filtering, it still takes about 9 seconds to complete the sync.  Raising this to 30 seconds just to avoid any transient timeout issues.